src/map/attack.cpp

```diff
@@ -446,6 +446,24 @@ bool CAttack::CheckCounter()
     return m_isCountered;
 }
 
+bool CAttack::IsCovered()
+{
+    return m_isCovered;
+}
+
+bool CAttack::CheckCover()
+{
+    if (m_attackRound->GetCoverEntity() != nullptr && (m_attackRound->GetCoverEntity()->isAlive()))
+    {
+        m_isCovered = true;
+    }
+    else {
+        m_isCovered = false;
+    }
+
+    return m_isCovered;
+}
```

Comment:
what's the need for having both of these checks?

Reply:
One performs the actual check, one just retrieves the value.